src/__tests__/http-test.ts

```diff
@@ -2269,5 +2269,32 @@ function runTests(server: Server) {
       expect(response.type).to.equal('application/json');
       expect(response.text).to.equal('{"data":{"test":"Hello World"}}');
     });
+
+    it('contains a default query within GraphiQL', async () => {
```

Comment:
this it statement should be about subscriptions, correct?

Reply:
right, I changed its statement and move this test case within the graphiql section as well.

Reply:
I added the sample implementation and usage of ```subscriptionEndpoint``` in graphiql options in README.  graphiql itself doesn't work as service platform which is already explained well in the README. Hence I resolved this conversation.